eq-author/src/App/QuestionnairesPage/withCreateQuestionnaire.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable graphql/template-strings */
```

Comment:
Is this needed?

Reply:
No, it was left over from trying different ways to fix the linting issue last week. Now removed.